firmware/user/modes/mode_mtype.c

```diff
@@ -2043,4 +2110,9 @@ void ICACHE_FLASH_ATTR playerDeath (void) {
         // start spawn invincibility.
         mType->player.invincibilityCountdown = PLAYER_INVINCIBILITY_TIME;
     }
+}
+
+// bitwise even check, should be faster than mod operator
+uint8_t ICACHE_FLASH_ATTR isEven (uint32_t n) {
```

Comment:
I don't think this is faster than mod, and it would definitely be faster as `inline`. You can check it in godbolt and count the resulting instructions.